### PR TITLE
Admin UI: Loading stops before data is shown

### PR DESCRIPTION
**Pull request by daniel-almeida-konkconsulting**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [X] I ensured that the PR title is good enough for the changelog.
-   [X] I labeled the PR.

### CODE REVIEW

**Review by tnotheis — COMMENTED** [review]
Now we don't have a loading indicator at all?

**Comment by daniel-almeida-konkconsulting** [comment]
> Now we don't have a loading indicator at all?

Of course we do, what makes you say that? ❓

**Comment by tnotheis** [comment]
Let me rephrase the question: why did you remove all these `loading` properties?

**Comment by daniel-almeida-konkconsulting** [comment]
> Let me rephrase the question: why did you remove all these `loading` properties?

I only removed the one that was unused.
The other components still have their loading property, the only behaviour change was that the animation is now only contingent on the main request (e.g., identities overview), as opposed to being toggle for items such as the filters.

**Review by tnotheis — APPROVED** [review]
